tools/clang/test/SemaHLSL/bitfields.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -HV 2021 -fsyntax-only -ffreestanding -verify %s
+// RUN: %dxc -Tlib_6_3 -HV 2021 -verify %s
```

Comment:
This shader looks like it is intended to be a vertex shader.

Reply:
Fixed.